cmd/ocsp-responder/main.go

```diff
@@ -220,8 +220,13 @@ as generated by Boulder's single-ocsp command.
 		KillTimeout: killTimeout,
 		Stats:       metrics.NewFBAdapter(scope, clock.Default()),
 	}
-	err = httpdown.ListenAndServe(srv, hd)
+	hdSrv, err := hd.ListenAndServe(srv)
 	cmd.FailOnError(err, "Error starting HTTP server")
+
+	go cmd.CatchSignals(logger, func() { _ = hdSrv.Stop() })
```

Comment:
This won't execute until after ListenAndServe exits, which means the program is done. This should be moved above ListenAndServe, and the `<-forever` below deleted.